vpr/src/base/read_options.cpp

```diff
@@ -2000,11 +2000,19 @@ argparse::ArgumentParser create_arg_parser(std::string prog_name, t_options& arg
 
     route_timing_grp.add_argument(args.astar_fac, "--astar_fac")
         .help(
-            "Controls the directedness of the the timing-driven router's exploration."
+            "Controls the directedness of the timing-driven router's exploration."
             " Values between 1 and 2 are resonable; higher values trade some quality for reduced run-time")
         .default_value("1.2")
         .show_in(argparse::ShowIn::HELP_ONLY);
 
+    route_timing_grp.add_argument(args.router_profiler_astar_fac, "--router_profiler_astar_fac")
+        .help(
+            "Controls the directedness of the timing-driven router's exploration"
+            " when doing router profiling."
```

Comment:
I think you should explain what "router profiling" means.  Does this mean during calculation of point-to-point delays for use during placement delay estimation & optimization?  Does it mean calculation of terms used by the router lookahead?  Does it mean both?

Reply:
Sure, makes sense.
One thing I see is that the RouterDelayProfiler is used only in the timing_place_lookup to generate the place delay matrix actually. I am unsure if its original purpose was to serve as a general interface to run the router, or if it was intended only for the delay matrix.

Reply:
I think we only ever used it for the place delay matrix.  One day there is probably some code that can be unified between the lookahead builder and the place delay matrix (use a single Djikstra expansion and get all the info perhaps ...).  But for now just saying what we use this parameter for would be good.